projects/kit/components/input-password/input-password.directive.ts

```diff
@@ -23,12 +30,14 @@ export class TuiInputPasswordDirective
         this.input = input;
     }
 
-    public ngDoCheck(): void {
-        if (
-            this.host.nativeFocusableElement &&
-            tuiIsInput(this.host.nativeFocusableElement)
-        ) {
-            this.host.nativeFocusableElement.type = this.host.inputType;
-        }
+    public ngAfterViewInit(): void {
```

Comment:
can we use `afterNextRender(() => {})` in constructor?

Reply:
From the documentation:

> Components are not guaranteed to be hydrated before the callback runs.

Are we sure we want to use afterNextRender? I believe using the ngAfterViewInit hook is sufficient, as it guarantees that the necessary input will be initialized.

Reply:
sure